### PR TITLE
Ensure setup as specified in README works

### LINKED ISSUE #315: README instructions throw errors
**Describe the bug**
When following the instructions on the README the project doesn't run as the instructions say. Instead a lot of errors are thrown.

**To Reproduce**

1. Follow README Instructions from the beginning
2. Errors at `npm run storybook` and `npm run test`

**Expected behavior**

There should be no errors and the project should run.

**Additional context**
When running an `npm run build` before running `npm run storybook` it will work!

- [x] Initially labelled with ["bug"](https://github.com/bbc/psammead/labels/bug)

### PR DESCRIPTION
**Pull request by twinlensreflex**
Resolves #315  

**Overall change:** 
README is now consistent.

**Code changes:**
There is now a `postinstall` script which will build the project so that storybook runs without errors when cloning the project initially. The README has also been reordered so that the package linking is in a more logical place.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +65,18 @@ NB, we've defined global styles (normalize, box-sizing, Reith font) in the [Stor
 npm run build
 ```
 
+## :computer: Developing with Psammead
+
+When making changes to a package locally if you want to pull those changes into another psammead package then the following command will create the required symlinks for you.
+
+### :link: Link packages locally
+
+Run the following command to link all psammead packages up regardless of dependancy version:
```
Comment (jamesdonoh) [inline README.md]: Typo on `dependancy` (pre-existing)

**Review by jamesdonoh — DISMISSED** [review]
Looks good, one tiny pedantic comment.

File: README.md [hunk 2]
```diff
@@ -25,22 +25,34 @@ Psammead packages are split into:
 git clone git@github.com:bbc/psammead.git
 ```
 
-### :hammer: Setup Local Environment
+### Check npm and nvm versions
```
Comment (dr3) [inline README.md]: We actually have a script to check this for you :) Perhaps we could make `sh scripts/enforceVersions.sh` a preinstall step? :)

Comment (twinlensreflex) [reply README.md]: https://github.com/bbc/psammead/blob/0ac0405fb22068dc2e05cf56636af82257a99772/package.json#L15 added :) thank you!

**Review by pjlee11 — DISMISSED** [review]
LGTM 👍

File: README.md [hunk 3]
```diff
@@ -73,6 +65,18 @@ NB, we've defined global styles (normalize, box-sizing, Reith font) in the [Stor
...
+## :computer: Developing with Psammead
+
+When making changes to a package locally if you want to pull those changes into another psammead package then the following command will create the required symlinks for you.
+
+### :link: Link packages locally
+
+Run the following command to link all psammead packages up regardless of dependency version:
+
+```
+npm install:packages:link
```
Comment (dr3) [inline README.md]:
```suggestion
npm run install:packages:link
```
Got lost in the latest merge

**Review by dr3 — APPROVED** [review]
👍 LGTM
Was a tad worried about the post install happening before the packages were installed, but tested locally with this and it works great 
```
cd .. && rm -rf psammead && git clone git@github.com:bbc/psammead.git && cd psammead && git checkout install-bug && npm run ci:packages && npm run storybook
```

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Comment by PriyaKR** [comment]
LGTM.

**Pull request merged**